diff --git a/lib/doc/image.js b/lib/doc/image.js
index beefae8396b7ef3827450efcd2d96d4560054795..17c63cdc4a073a8accbd1333eb7a2879dca97c6a 100644
--- a/lib/doc/image.js
+++ b/lib/doc/image.js
@@ -24,6 +24,7 @@ class Image {
             br: this.range.br && this.range.br.model,
             ext: this.range.ext,
             editAs: this.range.editAs,
+            behindDoc: this.range.behindDoc,
           },
         };
       default:
@@ -50,6 +51,7 @@ class Image {
           ext: range.ext,
           editAs: range.editAs,
           hyperlinks: hyperlinks || range.hyperlinks,
+          behindDoc: range.behindDoc,
         };
       }
     }
diff --git a/lib/xlsx/xform/drawing/two-cell-anchor-xform.js b/lib/xlsx/xform/drawing/two-cell-anchor-xform.js
index bc567e3b7042656b784ada5873de486648c82042..74e749640fab2aea52a3f75e73dde11809f2a462 100644
--- a/lib/xlsx/xform/drawing/two-cell-anchor-xform.js
+++ b/lib/xlsx/xform/drawing/two-cell-anchor-xform.js
@@ -25,7 +25,9 @@ class TwoCellAnchorXform extends BaseCellAnchorXform {
   }
 
   render(xmlStream, model) {
-    xmlStream.openNode(this.tag, {editAs: model.range.editAs || 'oneCell'});
+    const attrs = {editAs: model.range.editAs || 'oneCell'}
+    if (model.range.behindDoc) attrs.behindDoc = '1'
+    xmlStream.openNode(this.tag, attrs);
 
     this.map['xdr:from'].render(xmlStream, model.range.tl);
     this.map['xdr:to'].render(xmlStream, model.range.br);
